src/boardToJSON.ts

```diff
@@ -0,0 +1,20 @@
+import {GameState} from "./gameState"
+
+export class BoardToJSON{
+    public static convert(board:number[][]):{"ShipPositions":Array<any>,"MyShots": Array<any>,"OpponentsShots": Array<any>} {
+        let ans:{"ShipPositions":Array<any>, "MyShots":Array<any>, "OpponentsShots":Array<any>} = {"ShipPositions":[],"MyShots":[], "OpponentsShots":[]  };
```

Comment:
I'm really not sure that I like this setup for types at all. Why couldn't we have made this a class with shipPositions, myShots and opponentsShots? It would massively simplify this file.